lib/components/regions_map/interactor/view_models/regions_map_bloc.dart

```diff
@@ -104,7 +104,7 @@ class RegionsMapBloc extends Bloc<RegionsMapEvent, RegionsMapState> {
       final result = await GetPlacesFromCoordinatesUseCase()
           .run(GetPlacesFromCoordinatesUseCase.input(lat: event.pickedLat, lng: event.pickedLong));
       if (result.isError) {
-        emit(state.copyWith(pageState: PageState.failure));
+        // No address information found for supplied coordinates.
```

Comment:
so we do nothing? 
Is this expected?

Reply:
Well those address not found usually are from the sea. So we only ignore those not found places.